Mariia/lab6/service.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: weblab6
+  namespace: lab
+spec:
+  type: ClusterIP
+  ports:
+    - port: 8080
```

Comment:
It will not work. Because Nginx works on port 80 by default.
So You have to use target port 80 or change nginx configuration in deployment.

Reply:
Nginx default config
https://www.nginx.com/resources/wiki/start/topics/examples/full/